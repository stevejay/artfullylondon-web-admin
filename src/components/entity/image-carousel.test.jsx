import React from 'react'
import { shallow } from 'enzyme'

import EntityImageCarousel from '_src/components/entity/image-carousel'

it('should render correctly', () => {
  const wrapper = shallow(
    <EntityImageCarousel
      images={[
        {
          id: 'some-id',
          ratio: 2,
          copyright: 'The Copyright',
          dominantColor: 'AAAAAA'
        }
      ]}
    />
  )

  expect(wrapper).toMatchSnapshot()
})
