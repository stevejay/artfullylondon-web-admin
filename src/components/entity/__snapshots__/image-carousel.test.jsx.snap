// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div>
  <ImageGallery
    autoPlay={false}
    disableArrowKeys={false}
    disableSwipe={false}
    disableThumbnailScroll={false}
    flickThreshold={0.4}
    indexSeparator=" / "
    infinite={true}
    items={
      Array [
        Object {
          "copyright": "The Copyright",
          "original": "https://images.test.com/so/me/some-id/750x.jpg",
          "ratio": 2,
        },
      ]
    }
    lazyLoad={true}
    onSlide={[Function]}
    preventDefaultTouchmoveEvent={false}
    renderFullscreenButton={[Function]}
    renderItem={[Function]}
    renderLeftNav={[Function]}
    renderPlayPauseButton={[Function]}
    renderRightNav={[Function]}
    showBullets={false}
    showFullscreenButton={false}
    showIndex={false}
    showNav={true}
    showPlayButton={false}
    showThumbnails={false}
    slideDuration={450}
    slideInterval={3000}
    slideOnThumbnailHover={false}
    startIndex={0}
    stopPropagation={false}
    swipeThreshold={30}
    swipingTransitionDuration={0}
    thumbnailPosition="bottom"
    useBrowserFullscreen={true}
  />
  <ImageCredit
    credit="The Copyright"
  />
</div>
`;
